20_usage/10_web_interface.txt

```diff
@@ -23,30 +23,68 @@ image::./images/rudder-home.png[Home menu]
 
 ==== Node Management
 
-In the Node Management section, you will find the validation tool for new
+In the Node Management section, you will find, the list of all Nodes, the validation tool for new
```

Comment:
i think the first added comma is too much, it should read
In the Node Management section, you will find the list of all Nodes, the validation